docs/document_store_schema.md

```diff
@@ -0,0 +1,384 @@
+# Document Store Schema
+
+## Event
```

Comment:
This is kind of a preference thing, but I think it'd be nice to italicize the name of the table? The size difference in font between markdown heading level 2 and 3 isn't that big, so I think italicizing or bolding would make the schema easier to read when scrolling down.

Reply:
I will italicize all the H3s :+1: